lambdas/webhook_processor/webhook_processor.py

```diff
@@ -0,0 +1,45 @@
+import base64
+import json
+import os
+
+from botocore.vendored import requests
+
+import logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+logger.info('Loading function')
+
+def lambda_handler(event, context):
+
+    url = os.environ.get('TARGET_URL')
```

Comment:
There are going to be two endpoints we care about:
* $url/ghprbhook
* $url/github-webhook (for webhooks sent when merging to master)